Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM python:3.10.14-slim-bullseye
+
+LABEL    software="spades" \ 
+    author="Sebastian Bassi <sebastian@toyoko.io>" \
+    base_image="python:3.10.14-slim-bullseye" \ 
+    container="spades" \ 
+    about.summary="genome assembler for single-cell and isolates data sets" \ 
+    about.home="https://ablab.github.io/spades/" \ 
+    software.version="4.0.0" \ 
+    upstream.version="4.0.0" \ 
+    version="1" \ 
+    extra.identifiers.biotools="spades" \ 
+    about.copyright="2011-2014 Saint-Petersburg Academic University" \ 
```

Comment:
This is not accurate.

Reply:
please check again, just made changes to address these issues

Reply:
I would probably suggest you to build in the same way as release snapshots are being built: https://github.com/ablab/spades/blob/main/.github/workflows/release.yml#L53

Reply:
I've just tried this method, got the binaries but the spades.py test does not end in bin directory.

Here is what I tried:

```
cmake -B build/ -S src/ -DSPADES_USE_NCBISDK=ON -DSPADES_ENABLE_PROJECTS=all
cmake --build build/ -j16
```

after this, binaries are in bin, but not spades.py

Reply:
You'd also need to to `cmake --install`. Alternatively, just pass `-DSPADES_USE_NCBISDK=ON -DSPADES_ENABLE_PROJECTS=all` to `spades_compile.sh`.

Reply:
will try it tomorrow

Reply:
done, updated the Dockerfile and tested it